ios/RN/RNCameraManager.m

```diff
@@ -371,11 +371,31 @@ + (NSDictionary *)faceDetectorConstants
 
 RCT_EXPORT_METHOD(checkVideoAuthorizationStatus:(RCTPromiseResolveBlock)resolve
                   reject:(__unused RCTPromiseRejectBlock)reject) {
-    __block NSString *mediaType = AVMediaTypeVideo;
-    
-    [AVCaptureDevice requestAccessForMediaType:mediaType completionHandler:^(BOOL granted) {
-        resolve(@(granted));
-    }];
+    if ([[NSBundle mainBundle].infoDictionary objectForKey:@"NSCameraUsageDescription"] != nil) {
```

Comment:
@sibelius What is your opinion on this change?

I noticed that many users report app crashes and have issues with providing xcode logs and similar.

IMHO this approach is more helpful for new users as it shows a warning with whats wrong instead of crashing the app.

Reply:
I prefer this approach

Better to provide a fix path for users

Reply:
I try to do the same for android once I have access to an android device and can test it